### PR TITLE
Update Doctrine2.php

### PR DESCRIPTION
**Pull request by ThomasLandauer**
See https://github.com/Codeception/Codeception/issues/5286

### CODE REVIEW

**Comment by Naktibalda** [comment]
Sorry, Codeception has moved on, please make a new pull request to https://github.com/Codeception/module-doctrine2/ .

**Comment by ThomasLandauer** [comment]
Done :-)
https://github.com/Codeception/module-doctrine2/pull/1

File: src/Codeception/Module/Doctrine2.php [hunk 1]
```diff
@@ -37,8 +37,21 @@
  *
  * This will use static method of `MyDb::createEntityManager()` to establish the Entity Manager.
  *
- * By default, the module will wrap everything into a transaction for each test and roll it back afterwards. By doing this
- * tests will run much faster and will be isolated from each other.
+ * By default, the module will wrap everything into a transaction for each test and roll it back afterwards
+ * (this is controlled by the `cleanup` setting).
+ * By doing this, tests will run much faster and will be isolated from each other.
+ *
```
[the remark this answers is not in the document]
Comment (ThomasLandauer) [reply src/Codeception/Module/Doctrine2.php]: Fixed (I hope :-)

**Comment by ThomasLandauer** [comment]
Merged at https://github.com/Codeception/module-doctrine2/pull/2

**Pull request closed without merging**